2019-09-18 13:39:16.990 [task-1] ERROR c.o.cloud.system.socket.utils.DataSegmentParseUtil - 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_201909' doesn't exist
### The error may exist in com/osen/cloud/model/data/DataHistoryMapper.java (best guess)
### The error may involve com.osen.cloud.model.data.DataHistoryMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO data_history_201909  ( date_time, purifier_flag, nmhc_flag, nmhc, pm_flag, lampblack_flag, lampblack, device_no, pm, fan_flag )  VALUES  ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_201909' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_201909' doesn't exist
2019-09-18 14:52:42.666 [task-1] ERROR c.o.cloud.system.socket.utils.DataSegmentParseUtil - 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
### The error may exist in com/osen/cloud/model/data/DataHistoryMapper.java (best guess)
### The error may involve com.osen.cloud.model.data.DataHistoryMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO data_history_20199  ( date_time, purifier_flag, nmhc_flag, nmhc, pm_flag, lampblack_flag, lampblack, device_no, pm, fan_flag )  VALUES  ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
2019-09-18 14:56:02.586 [task-1] ERROR c.o.cloud.system.socket.utils.DataSegmentParseUtil - 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_minute_20199' doesn't exist
### The error may exist in com/osen/cloud/model/data/DataMinuteMapper.java (best guess)
### The error may involve com.osen.cloud.model.data.DataMinuteMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO data_minute_20199  ( date_time, lampblack_min, pm_min, nmhc, pm_flag, lampblack_flag, lampblack, device_no, nmhc_min, fan_flag, purifier_flag, nmhc_flag, lampblack_max, pm_max, pm, nmhc_max )  VALUES  ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_minute_20199' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_minute_20199' doesn't exist
2019-09-18 14:56:03.859 [task-2] ERROR c.o.cloud.system.socket.utils.DataSegmentParseUtil - 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
### The error may exist in com/osen/cloud/model/data/DataHistoryMapper.java (best guess)
### The error may involve com.osen.cloud.model.data.DataHistoryMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO data_history_20199  ( date_time, purifier_flag, nmhc_flag, nmhc, pm_flag, lampblack_flag, lampblack, device_no, pm, fan_flag )  VALUES  ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
2019-09-18 15:08:30.290 [task-1] ERROR c.o.cloud.system.socket.utils.DataSegmentParseUtil - 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
### The error may exist in com/osen/cloud/model/data/DataHistoryMapper.java (best guess)
### The error may involve com.osen.cloud.model.data.DataHistoryMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO data_history_20199  ( date_time, purifier_flag, nmhc_flag, nmhc, pm_flag, lampblack_flag, lampblack, device_no, pm, fan_flag )  VALUES  ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_20199' doesn't exist
2019-09-18 15:09:12.615 [task-1] ERROR c.o.cloud.system.socket.utils.DataSegmentParseUtil - 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_201909' doesn't exist
### The error may exist in com/osen/cloud/model/data/DataHistoryMapper.java (best guess)
### The error may involve com.osen.cloud.model.data.DataHistoryMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO data_history_201909  ( date_time, purifier_flag, nmhc_flag, nmhc, pm_flag, lampblack_flag, lampblack, device_no, pm, fan_flag )  VALUES  ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_201909' doesn't exist
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Table 'osen-dev.data_history_201909' doesn't exist
